cmd/solana-exporter/utils.go

```diff
@@ -186,3 +187,65 @@ func CountVoteTransactions(block *rpc.Block) (int, error) {
 	}
 	return voteCount, nil
 }
+
+// BoolToFloat64 converts a boolean to either 1.0 or 0.0
+func BoolToFloat64(b bool) float64 {
+	if b {
+		return 1
+	} else {
```

Comment:
It's generally preferred to not use an else branch unless necessary. The "then" block returns, so you can avoid this.

Reply:
I always make the same review comment when doing this in python. Less branches == less context to hold in your head when reading the code.